### PR TITLE
OOP school library: unit tests

### PR DESCRIPTION
**Pull request by svitalis123**
# OOP school library: unit tests
## I accomplished

- [ ] Added unit tests for all the methods in all the classes: Book Person, Student, Teacher, Classroom, Rental 
- [ ] Created branch

### CODE REVIEW

File: spec/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    book = Book.new 'benardo', 'bello'
+    it 'expect title to be benardo' do
+      expect(book.title).to eql 'benardo'
+    end
+
+    it 'edxpect author to be bello' do
+      expect(book.author).to eql 'bello'
+    end
+  end
+end
```
Comment (DammyShittu) [inline spec/book_spec.rb]: - [x] Kindly add test for the `rent_book` method in the Book class.

File: spec/person_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  context 'when an instance created and parameter passed' do
+    person = Person.new(29, 'mankush')
+    it 'expect age to be 29' do
+      expect(person.age).to eql 29
+    end
+    it 'expect name to be mankush' do
+      expect(person.name).to eql 'mankush'
+    end
+  end
+end
```
Comment (DammyShittu) [inline spec/person_spec.rb]: - [x] Kindly add tests for the `can_use_services?` and `rent_book` methods in the Person class.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @svitalis123 and @samuelmoiwa,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/person_spec.rb (hunk 2 above)
Comment (svitalis123) [reply spec/person_spec.rb]: Thank you I have resolved this

**Review by danmainah — APPROVED** [review]
Hi @svitalis123 and @samuelmoiwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

- [ ] [optional] It would be nice if you remove the microverse logo from your readme also kindly add the key features and future features section in your readme file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**